crates/testing/tests/tests_1/consensus_task.rs

```diff
@@ -68,19 +66,18 @@ async fn test_consensus_task() {
     }
 
     // Run view 1 (the genesis stage).
-    let view_1 = TestScriptStage {
-        inputs: vec![
+    let view_1 = TestScriptStage::from_inputs_and_outputs(
+        vec![
             QuorumProposalRecv(proposals[0].clone(), leaders[0]),
             DaCertificateRecv(dacs[0].clone()),
             VidShareRecv(vid_share(&vids[0].0, handle.public_key())),
         ],
-        outputs: vec![
+        vec![
             exact(ViewChange(ViewNumber::new(1))),
             quorum_proposal_validated(),
             exact(QuorumVoteSend(votes[0].clone())),
         ],
-        asserts: vec![],
-    };
+    );
```

Comment:
This might just be a personal thing, but I find this somewhat less clear read with `from_inputs_and_outputs` compared to the constructor because the `inputs:` and `outputs:` labels are lost

Reply:
Yeah I think you're right as well, I'll roll this change back.